cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -447,7 +447,17 @@ values for Cloudera may vary from the above appendix:
     :start-after: .. _cloudera-starting-services-java-heapmax:
     :end-before: .. end_of_list
 
-At this point, the CDAP installation is configured and is ready to be installed. Review
+**The CDAP system user:** As Hadoop resolves users at the NameNode, the ``cdap`` user must
```

Comment:
I think this section would be more appropriate under "Preparing the cluster", and more consistent with the other installation methods.  Let me know if you disagree.  There is already a mention of the cdap user there (line 87) that should at least link to this section.